charts/hono/values.yaml

```diff
@@ -1724,82 +1724,68 @@ kafka:
   # Set to false for productive setups. Topic management then needs to be provided externally.
   autoCreateTopicsEnable: true
   deleteTopicEnable: true
-  auth:
-    clientProtocol: "sasl_tls"
-    sasl:
-      jaas:
-        clientUsers:
-        - "hono"
-        clientPasswords:
-        - "hono-secret"
-        zookeeperUser: "zookeeperUser"
-        zookeeperPassword: "zookeeperPassword"
-    tls:
-      type: "pem"
-      pemChainIncluded: true
-      existingSecrets:
-      # Evaluated in kafka chart via tpl; kafka chart full name used here!
-      - '{{ include "hono.fullname" . }}-example-keys'
 
-  service:
-    ports:
-      client: 9092
-      internal: 9093
-      external: 9094
+  broker:
+    automountServiceAccountToken: true
+
+  controller:
+    replicaCount: 1
+    automountServiceAccountToken: true
+    extraConfig: |
+      group.initial.rebalance.delay.ms=0
+      offsets.topic.replication.factor=1
+      transaction.state.log.min.isr=1
+      transaction.state.log.replication.factor=1
+
+  listeners:
+    client:
+      protocol: "SASL_SSL"
+    external:
+      protocol: "SASL_SSL"
+
+  sasl:
+    client:
+      users:
+      - "hono"
+      passwords:
+      - "hono-secret"
+
+  tls:
+    type: "PEM"
+    pemChainIncluded: false
+    # Evaluated in kafka chart via tpl; kafka chart full name used here!
+    existingSecret: '{{ include "hono.fullname" . }}-example-keys'
 
-  replicaCount: 1
   # Do not write data to a PersistentVolume by default.
   # When setting this to true, make sure to also set "kafka.zookeeper.persistence.enabled" to true and
   # make sure that Kafka is stopped cleanly using SIGTERM _before_ Zookeeper is stopped.
   # Otherwise you will most likely run into issues like
   # https://stackoverflow.com/questions/39759071/error-while-starting-kafka-broker
-  persistence:
-    enabled: false
+#  persistence:
+#    enabled: false
```

Comment:
It looks like there is no top-level `persistence` property anymore in the Kafka-chart. 
Also the mentioning of the `kafka.zookeeper.persistence.enabled` setting above could be rather confusing, now that KRaft mode is used by default.